extensions/parquet/table/src/main/java/io/deephaven/parquet/table/metadata/TableInfo.java

```diff
@@ -61,11 +61,16 @@ public final Map<String, ColumnTypeInfo> columnTypeMap() {
     }
 
     /**
-     * @return The Deephaven release version when this metadata format was defined
+     * @return The Deephaven release version used to write the parquet file
      */
     @Value.Default
     public String version() {
-        return "0.4.0";
+        final String version = TableInfo.class.getPackage().getImplementationVersion();
+        if (version == null) {
+            // For unit tests
+            return "0.dev.0";
```

Comment:
When will this happen? What's special about the unit test environment? Seems like this would be bad in a lot of cases.
Would "unknown" satisfy the test requirement?

Reply:
This is for anytime the code is run from class files as opposed to jars. I think "unknown" is better in that case as well too.